lib/defaults/defaults.go

```diff
@@ -46,6 +46,9 @@ const (
 	// run behind an environment/firewall which only allows outgoing connections)
 	SSHProxyTunnelListenPort = 3024
 
+	// Default port to listen locally and respond to HTTP GET health check requests on
+	HealthcheckListenPort = 3027
```

Comment:
This doesn't actually seem to be used - as in, when I run the client I don't see anything binding to `127.0.0.1:3027` as I would expect. I presume that this is a quirk of the multiplexer?

Reply:
I don't have all the context, but the way I read the listener setup stuff in `lib/service/service.go` is that yeah, its going to either be multiplexing *or* be using that port.